app/views/projects/show.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1><%= @project.name %></h1>
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
+<p>Number of Contestants: <%= @project.contestants.count %></p>
+<p>Average Age of Contestants: <%= @project.contestants.average(:age) %></p>
```

Comment:
Using `.average` in the view is an MVC infraction since we're asking the database directly for information. This should live in a model method, and then test it in the model tests.